widgets/device.js

```diff
@@ -1493,6 +1499,7 @@ $.widget("sv.device_uzsugraph", $.sv.device_uzsu, {
     chart.redraw();
 
     self._plotNowLine();
+    Highcharts.seriesTypes.scatter.prototype.getPointSpline = Highcharts.seriesTypes.line.prototype.getPointSpline;
```

Comment:
I would prefer that the original getPointSpline of scatter be saved before overwriting it on line 1387 and then reapplied here.
(currently `Highcharts.seriesTypes.line.prototype.getPointSpline` as well as `Highcharts.seriesTypes.scatter.prototype.getPointSpline` are `undefined` anyway, but obviously this could be changed in Highcharts in future)

Reply:
Sure. Can you add that?